### ISSUE TITLE
Text selection

### ISSUE THREAD

**Issue by samuelmeuli** [samuelmeuli/glance#13]
Users might want to select and copy text in Quick Look previews.

_Requested [on Reddit](https://www.reddit.com/r/macapps/comments/gkw623/glance_allinone_quick_look_plugin/fququha?utm_source=share&utm_medium=web2x)._

**Comment by samuelmeuli** [comment]
This isn't possible in macOS's default previews. [It used to be possible to enable it, though](https://apple.stackexchange.com/questions/205574/copy-text-from-quick-look-in-el-capitan-or-sierra). Would've been nice if Glance could've read that setting to determine its behavior.

**Comment by VKondakoff** [comment]
Yes, lack of the ability to copy text is one of the most annoying default previews restriction. I hope Glance will be able to copy text to the clipboard.

**Comment by samuelmeuli** [comment]
Until now, Glance's behavior was that text can be selected, but not copied. Pressing `⌘` + `C` copies the file name instead of the selection. I haven't yet figured out how to allow copying text inside Quick Look and have my doubts that it's even possible.

To avoid confusion, I've disabled text selection for now (c274bae). If anyone with experience in macOS development knows if the behavior can be changed, I'd be happy to hear their thoughts :)

**Comment by mrousavy** [comment]
I don't think interaction with the quicklook content is intended behavior, as it really should just be a quick-look. Not a quick-edit. Nevertheless, awesome plugin, really enjoying my productivity boost!
